src/signalo/core/tests.py

```diff
@@ -130,3 +131,41 @@ def test_with_poleserializer(self):
         with open(os.path.join(self.path, f"{name}.txt"), "w") as fh:
             stats = pstats.Stats(path_to_bin, stream=fh)
             stats.print_stats()
+
+
+class TestBasicAuth(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        call_command("populate_vl")
+        call_command("populate_signs_poles")
+        call_command("populate_users")
+
+        cls.demo_viewer = User.objects.get(username="demo_viewer")
+        cls.admin = User.objects.get(username="admin")
+
+    def tearDown(self):
+        self.client.force_authenticate(user=None)
+
+    def test_auth_as_demo_viewer(self):
+        collections_from_anonymous = self.client.get(
+            "/oapif/collections", format="json"
+        ).json()
+        self.client.force_authenticate(user=self.demo_viewer)
+        collection_response = self.client.get("/oapif/collections", format="json")
+        oapif_response_code = self.client.get("/oapif/").status_code
+
+        self.assertEqual(oapif_response_code, 200)
+        self.assertEqual(collection_response.status_code, 200)
+        self.assertEqual(len(collection_response.json()), len(collections_from_anonymous))
```

Comment:
The latest job results show that this has succeeded. From my understanding this entails that demo_viewer has no issue accessing the root view or the collections view, from the point of view of DRF.